From dd925de72b46a1982cb5515c2e90d67b1744f0bf Mon Sep 17 00:00:00 2001
From: Max Horn <max@quendi.de>
Date: Sun, 26 Dec 2021 17:28:46 +0100
Subject: [PATCH] Don't set HAVE_SSP unconditionally on aarch64

There is no plausible reason for this. And indeed, this patch fixes
compilation for the libjulia_jll builder on Yggdrasil which currently fails
due to the absence of libssp.

The removed line was introduced in PR #41936 but without any justification
that I could discern, so it might have just slipped in there accidentally.
---
 Make.inc | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Make.inc b/Make.inc
index 9706b1f929..b73d9033ca 100644
--- a/Make.inc
+++ b/Make.inc
@@ -891,7 +891,6 @@ OPENBLAS_DYNAMIC_ARCH:=0
 OPENBLAS_TARGET_ARCH:=ARMV8
 USE_BLAS64:=1
 BINARY:=64
-HAVE_SSP:=1
 ifeq ($(OS),Darwin)
 # Apple Chips are all at least A12Z
 MCPU:=apple-a12
-- 
2.34.0.7.g0ea906d205

